src/components/nodes/image_processing/Dimensions.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div/>
+</template>
+
+<script>
+import { BackendImage } from '../../../utils/ajax'
+
+export default {
+  spec: {
+    id: 'std::dims',
+    title: 'Dimensions',
+    icon: require('../../../assets/node_icons/node_dimensions.svg'),
+    inputs: [
+      {
+        id: 'image',
+        name: 'Image',
+        type: 'image'
+      }
+    ],
+    outputs: [
+      {
+        id: 'width',
+        name: 'Width',
+        type: 'number'
+      },
+      {
+        id: 'height',
+        name: 'Height',
+        type: 'number'
+      },
+      {
+        id: 'channels',
+        name: 'Channels',
```

Comment:
We should discuss whether this output is useful. With the current system, it will be 3 at all times.

Reply:
How come that neither RGBA nor grayscale formats are supported by the backend?

Reply:
Images are currently stored in the JPEG file format as that is uses an appropriate compression algorithm for photographs and the alpha channel should be unnecessary for paleography. Splitting channels into individual images and not storing them as JPEG may be unnecessarily complex, both for the backend as well as for the user.